lib/view/monthlyTaskView.dart

```diff
@@ -78,67 +114,43 @@ class _MonthlyTaskViewState extends State<MonthlyTaskView> {
                 setState(() {
                   _selectedDay = selectedDay;
                   _focusedDay = focusedDay;
-                });
-
-                // Call the fetchTodayTasks function without await
-                fetchTodayTasks(selectedDay);
-              }
-            },
-            onFormatChanged: (format) {
-              if (_calendarFormat != format) {
-                setState(() {
-                  _calendarFormat = format;
+                  fetchTodayTasks(selectedDay);
```

Comment:
I think this function shouldn't be defined here at all now that I look at it. `todayTasks` can be assigned by calling a db function that returns the new today tasks, and then assigning that value to the todayTasks with something like `todayTasks = functionCall`. This way, setState doesn't need to be called again within the function, as it seems weird to call setState within setState and to delegate setting todayTasks to another function. Ideally, database related functions should not change any external behavior.